python/sglang/srt/layers/quantization/fp8.py

```diff
@@ -432,7 +432,7 @@ def create_weights(
         from sglang.srt.layers.moe.fused_moe_triton import FusedMoeWeightScaleSupported
 
         if self.quant_config.is_checkpoint_fp8_serialized:
-            params_dtype = torch.float8_e4m3fn
+            params_dtype = torch.float8_e4m3fnuz if is_hip() else torch.float8_e4m3fn
```

Comment:
same problem here - check out the previous usage from `normalize_e4m3fn_to_e4m3fnuz`